### ISSUE TITLE
Physics update rate causes you to get caught up more often. 

### ISSUE THREAD

**Issue by NiamhHyx** [Lyall/RF5Fix#8]
I have noticed that while it really helps with the camera jitter, it also causes some small issues. 

If its around 60-120, its rare, but you can get stuck on little step ups in the game, and above 240 its happens almost every time.

Is this something that is possible for the mod to fix with in its self or something we have to wait for the developers to deal with?

**Comment by alliepet** [comment]
I am also experiencing this issue, not sure if I should revert to the version I updated from (0.5) until this is fixed.

**Comment by Lyall** [comment]
This does make sense given that increasing the update rate is essentially a band-aid fix. It definitely is something the developers could and should fix. 

There is the potential to add some kind of interpolation to the camera movement to smooth out the default 50hz update rate. There's a really good writeup by [@SuiMachine](https://suimachine.github.io/2022/07/07/Improving_Deadly_Premonition_2_using_HarmonyX_and_MelonLoader.html) on adding interpolation to character movement in Dead Premonition 2 with [Sui's Hack](https://github.com/SuiMachine/Deadly-Premonition-2---Sui-s-hack). Doing that kind of thing is probably beyond my meagre ability however. 

I think what I could do for now is to change the default of 240 to be set to your current screen refresh rate. This should make the update rate not be set far higher than it should be. For example if you play at 60fps you really only need the camera updating at 60hz from the default 50hz anyway.



> I am also experiencing this issue, not sure if I should revert to the version I updated from (0.5) until this is fixed.

Reverting wouldn't fix anything since it is directly related to the physics update rate as @NiamhHyx mentioned, which has been in RF5Fix since the start.

**Comment by NiamhHyx** [comment]
Makes sense to run it at your refresh rate, I did test 120 fps with 120 update rate and getting caught up felt about as uncommon as 60fps with 60 is. Cant test any higher than that though because my monitor is only 120hz. 

Maybe I'll start spamming xseeds discord and hope they see it so it doesn't end up having to be everyone else's problem for a long long time.
